Setup/Patch/Data/ExtendProductPatch.php

```diff
@@ -199,20 +199,6 @@ public function revert()
                     }
                 }
 
-                if (
```

Comment:
why are we removing this?

Reply:
Because we do not ever want this table deleted as it would erase any record of SP for these orders.